ddl/rollingback.go

```diff
@@ -17,14 +17,39 @@ import (
 	"github.com/pingcap/errors"
 	"github.com/pingcap/parser/ast"
 	"github.com/pingcap/parser/model"
+	"github.com/pingcap/parser/mysql"
 	"github.com/pingcap/tidb/kv"
 	"github.com/pingcap/tidb/meta"
 	"github.com/pingcap/tidb/util/logutil"
 	"go.uber.org/zap"
 )
 
+func updateNullColumnFlags(tblInfo *model.TableInfo, indexInfo *model.IndexInfo, flag uint) error {
+	nullCols, err := getNullColInfos(tblInfo, indexInfo)
+	if err != nil {
+		return errors.Trace(err)
+	}
+	for _, col := range nullCols {
+		col.Flag = flag
```

Comment:
Should use `col.Flag |= flag` and need to clean `PreventNullInsertFlag `.
![image](https://user-images.githubusercontent.com/26020263/68374296-94f85b00-017f-11ea-93eb-7adca344cbee.png)

Reply:
Greate catch! Done.